### PR TITLE
Cypress test for adding field in contact form in com_contact

### PR DESCRIPTION
**Pull request by rajputanuj31**
add a test adding field in contact form in com_contact.

### CODE REVIEW

File: tests/System/integration/site/components/com_contact/Field.cy.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+describe('Test that the Contact Field', () => {
+  it('can display a added field', () => {
```
Comment (sandewt) [inline tests/System/integration/site/components/com_contact/Field.cy.js]: an added field

File: tests/System/integration/site/components/com_contact/Field.cy.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    cy.db_createFieldGroup({ title: 'automated test field group', context: 'com_contact.mail' })
+      .then((id) => {
+        cy.db_createField({
+          group_id: id, context: 'com_contact.mail', type: 'checkboxes', fieldparams: '{"options":{"options0":{"name":"Facebook","value":""}}}',
```
Comment (laoneo) [inline tests/System/integration/site/components/com_contact/Field.cy.js]: Can you use also JSON.stringify as @sandewt did here https://github.com/joomla/joomla-cms/pull/40189/files#diff-68548e87f279990c390c725fa21aac37b6332827a365d9a7010cf45876559e99R4? Also please remove Facebook form the name and put something more generic like "test value".

File: tests/System/integration/site/components/com_contact/Field.cy.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+describe('Test that the Contact Field', () => {
+  it('can display a added field', () => {
+    cy.db_createFieldGroup({ title: 'automated test field group', context: 'com_contact.mail' })
+      .then((id) => {
+        cy.db_createField({
```
Comment (laoneo) [inline tests/System/integration/site/components/com_contact/Field.cy.js]: The same as in the test before, remove the surrounding {}.

File: tests/System/integration/site/components/com_contact/Field.cy.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+          group_id: id, context: 'com_contact.mail', type: 'checkboxes', fieldparams: '{"options":{"options0":{"name":"Facebook","value":""}}}',
+        });
+      });
+    cy.db_getUserId().then((id) => cy.db_createContact({ name: 'automated test contact 1', user_id: id }))
```
Comment (laoneo) [inline tests/System/integration/site/components/com_contact/Field.cy.js]: This must be wrapped in a .then from the call before.

File: tests/System/integration/site/components/com_contact/Field.cy.js (hunk 1 above)
Comment (rajputanuj31) [reply tests/System/integration/site/components/com_contact/Field.cy.js]: Done

File: tests/System/integration/site/components/com_contact/Field.cy.js (hunk 3 above)
Comment (rajputanuj31) [reply tests/System/integration/site/components/com_contact/Field.cy.js]: Done.

File: tests/System/integration/site/components/com_contact/Field.cy.js (hunk 2 above)
Comment (rajputanuj31) [reply tests/System/integration/site/components/com_contact/Field.cy.js]: Done.

File: tests/System/integration/site/components/com_contact/Field.cy.js (hunk 4 above)
Comment (rajputanuj31) [reply tests/System/integration/site/components/com_contact/Field.cy.js]: Done.

**Comment by laoneo** [comment]
Needs some mire default fields

**Comment by rajputanuj31** [comment]
Thanks a lot for merging it.

**Pull request merged**